package.json

```diff
@@ -48,6 +46,7 @@
     "@typescript-eslint/eslint-plugin": "^5.45.1",
     "@typescript-eslint/parser": "^5.45.1",
     "@vitejs/plugin-react": "^2.2.0",
+    "@walletconnect/chat-client": "^0.8.2",
```

Comment:
Accidental add as devDep?

Reply:
No - It's still used to build the types. Considering hard coding the type but not sure if it's the right approach

Reply:
Decided it wasn't worth the hassle and it's cleaner to hard code the types. But this required some additions.